### PR TITLE
fix: update dashboard wrong instance and add a legend for topology

### PR DESCRIPTION
**Pull request by Fine0830**
Fix dashboard wrong instance. Add a legend for topology. Update the condition of error cube.

Screenshot
Visit this page for the first time

legend

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/views/components/topology/chart/utils/nodeElement.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default (d3, graph, tool, funcs, tip) => {
     .attr('x', 2)
     .attr('y', 10)
     .attr('style', 'cursor: move;')
-    .attr('xlink:href', (d) => ((d.sla < 98 && d.isReal) ? icons.CUBEERROR : icons.CUBE));
+    .attr('xlink:href', (d) => (d.sla < 98 && d.isReal && d.cpm > 1 ? icons.CUBEERROR : icons.CUBE));
```
Comment (Fine0830) [inline src/views/components/topology/chart/utils/nodeElement.js:52]: Please check the condition of red cube @wu-sheng

Comment (wu-sheng) [reply src/views/components/topology/chart/utils/nodeElement.js:52]: Let's reduce the expectation of sla to 95. And we need to describe this clearly on the UI.
- Healthy
- Unhealthy(Successful Rate < 95% and Traffic > 1 call/min)

**Comment by wu-sheng** [comment]
Could you provide a new screenshot?

**Comment by Fine0830** [comment]
> Could you provide a new screenshot?

Here it is. https://github.com/apache/skywalking-rocketbot-ui/pull/409#issue-550350156

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**